### ISSUE TITLE
Investigate whether concrete playback can be used for test case generation

### ISSUE THREAD

**Issue by sanjit-bhat** [model-checking/kani#1525]
As recently suggested by @tedinski, concrete playback could potentially be used for test case generation. This is when a user is trying to generate a concrete test case that reaches a certain code path or triggers a certain failing condition.

I think this might be possible with how the feature is implemented right now. We might just need to test it more and add a section to the documentation for this use case.

**Comment by celinval** [comment]
I don't think this work is needed for the concrete playback stabilization though. Please let me know if you disagree.

**Comment by sanjit-bhat** [comment]
Agreed.

**Comment by celinval** [comment]
BTW, I'm assuming this is related to generating a test case that reaches a certain part of the code, like a branch condition or a line number.
